Automatically generate ngrok link.

mymadhavyadav16 opened the pull request:
Hi there,
I have modified and added an option in this tool to automatically generate **ngrok** URL which could be quite convenient for the user. Instead of downloading and running **ngrok** in background, now the user can simply run seeker.py in the following way to generate **ngrok** URL - 

```sudo python3 seeker.py -n```

This will generate ngrok URL which the user can later send to the target.

Also, I have modified the **README.md** file so you guys need not to worry about modifying it later :)
One more thing, I have added a folder **logs** which will resolve the issue #383.

Hope, it helps 😀❤

Thank you

thewhiteh4t commented:
@mymadhavyadav16 thanks for contributing!

1. ngrok is one service I used as an example for demo of the tool, there are many more

2. I dont want to specifically include ngrok in the tool because if ngrok or similar service is not working people come to complain here which is wrong

3. long time back I had added ngrok automation and realised that its better to launch it in a second terminal and removed this same feature

4. goal is not convenience its stability and I dont think its hard at all to open a second terminal

5. logs directory error is fixed in v1.2.8

mymadhavyadav16 commented:
> @mymadhavyadav16 thanks for contributing!
> 
> 1. ngrok is one service I used as an example for demo of the tool, there are many more
> 2. I dont want to specifically include ngrok in the tool because if ngrok or similar service is not working people come to complain here which is wrong
> 3. long time back I had added ngrok automation and realised that its better to launch it in a second terminal and removed this same feature
> 4. goal is not convenience its stability and I dont think its hard at all to open a second terminal
> 5. logs directory error is fixed in v1.2.8

Thanks for the insights...hope to contribute in future.
 😁👍
